composer.json

```
@@ -22,7 +22,7 @@
         }
     ],
     "require": {
-        "php": "^7.1",
+        "php": ">=7.1",
```

```suggestion
        "php": "^7.0 | ^8.0",
```
IMO We dont wonna support higer version like 9.0.

---

We had this discussion in symfony and also packagist now recommends using no upper boundries, because it cannot work, but it can work so 50/50

---

Yes, `>=` is recommended ; it avoids us all these pr when there will be php9.

---

But if in php9 will be more BC break with php7 and php8, than `>=7.1` cause batch of errors